lib/ffmpeg/transcoder.rb

```diff
@@ -19,11 +19,11 @@ def initialize(movie, output_file, options = EncodingOptions.new, transcoder_opt
 
       if options.is_a?(String)
         prefix_options = convert_prefix_options_to_string(transcoder_prefix_options)
-        @raw_options = "#{prefix_options}-i #{Shellwords.escape(@movie.path)} #{options}"
+        @raw_options = "#{prefix_options} #{EncodingOptions.new.send(:convert_inputs, @movie.paths)} #{options}"
```

Comment:
What's the reason for using `.send` here?

Reply:
`convert_inputs` is a private method, and if I make it non-private, it breaks the auto-selecting functionality of how `params` is calculated here:
https://github.com/Vidyard/rlovelett-ffmpeg/blob/9d0e7236b5899678fe7dda99db07ca9d0a0f38bb/lib/ffmpeg/encoding_options.rb#L11-L17

I decided against fighting it / hacking it for the single object, but if you have strong feelings here I can see about finding a workaround (or I'm open to suggestions!)

Reply:
oh no strong opinions, was mainly curious. Thanks

Reply:
Nah good call, this is a code smell -- fixed!